docs/user/installing/configuring-auth/auth-openshift.md

```
@@ -96,21 +98,27 @@ oc new-project my-org
 
 # Grant Flight Control operator permissions
-oc adm policy add-role-to-user flightctl-operator my-user -n my-org
+oc adm policy add-role-to-user flightctl-operator-<release> my-user -n my-org
 
 # Grant Flight Control viewer permissions
-oc adm policy add-role-to-user flightctl-viewer my-user -n my-org
+oc adm policy add-role-to-user flightctl-viewer-<release> my-user -n my-org
+```
+
+**Example:** If your Helm release name is `flightctl`, use:
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)